### PR TITLE
This is a new branch

### PR DESCRIPTION
**Pull request by Luzonero**
Hello what's up!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Discord-FalloutShelter
 
-# HELLO!!!
\ No newline at end of file
+# HELLO!!!
```
Comment (Zarasuto) [inline README.md:3]: BAKIT NAG LAGAY KA NG HELLO PANGIT YAN BURAHIN MO

**Review by Zarasuto — CHANGES_REQUESTED** [review]
pangit nakalagay

File: README.md (hunk 1 above)
Comment (Luzonero) [reply README.md:3]: OK SIGE..

**Review by Zarasuto — APPROVED** [review]
noice

**Pull request merged**